core/tests/puppeteer-acceptance-tests/utilities/user/logged-out-user.ts

```diff
@@ -2046,6 +2054,24 @@ export class LoggedOutUser extends BaseUser {
       showMessage('The donor box is visible on the about page.');
     }
   }
+
+  /**
+   * This function checks that user is on correct classroom page.
+   */
+  async validateCurrentClassroomPage(classroomName: string): Promise<void> {
+    await this.page.waitForSelector('.classroom-name');
```

Comment:
store '.classroom-name' as a constant at the start of this file.

Reply:
Done.